tools/taskcluster/tasks.py

```diff
@@ -84,41 +87,123 @@ def craft_release_task(self, tag):
             ['secrets:get:project/mobile/firefox-tv/tokens'],
             {
                 'public': artifact('directory', '/opt/firefox-tv/app/build/outputs/apk')
-            }
+            },
+            chain_of_trust=True,  # Needed for sign and push task verification
+        )
+
+    def craft_email_task(self, sign_task_id, push_task_id, tag):
+        # The "\\n" are hard-coded on purpose, since we don't want a newline in the string, but
+        # we do want the JSON to have the escaped newline.
+        # This email content is formatted with markdown by Taskcluster
+        content = 'Automation for this release is ready. Please: \\n' \
+                  f'* Download the APK and attach it to the [Github release](https://github.com/mozillamobile/firefox-tv/releases/tag/{tag})\\n' \
+                  '* [Deploy the new release on Amazon](https://developer.amazon.com/apps-and-games/console/app/amzn1.devportal.mobileapp.7f334089688646ef8953d041021029c9/release/amzn1.devportal.apprelease.4ca3990c43f34101bf5729543343747a/general/detail)'
+        script = f'''
+        curl -X POST "$TASKCLUSTER_PROXY_URL/notify/v1/email" -H "Content-Type: application/json" -d "$(cat <<EOM
+        {{
+            "address": "{NOTIFY_EMAIL_ADDRESS}",
+            "content": "{content}",
+            "link": {{
+                "href": "https://queue.taskcluster.net/v1/task/{sign_task_id}/artifacts/public/build/target.apk",
+                "text": "{tag} APK"
+            }},
+            "subject": "Release {tag} is ready for deployment"
+        }}
+        EOM
+        )"
+        '''
+
+        return self._craft_shell_task(
+            'Email that automation is complete',
+            script,
+            [f'notify:email:{NOTIFY_EMAIL_ADDRESS}'],
+            {},
+            dependencies=[sign_task_id, push_task_id],
         )
 
-    def _craft_shell_task(self, name, script, scopes, artifacts):
-        single_command = ' && '.join([line.strip() for line in script.split('\n') if line.strip()])
+    def _craft_shell_task(self, name, script, scopes, artifacts, *, dependencies=(), chain_of_trust=False):
+        trimmed_script = '\n'.join([line.strip() for line in script.split('\n') if line.strip()])
         bash_command = [
             '/bin/bash',
             '--login',
-            '-cx',
-            f'export TERM=dumb && {single_command}'
+            '-c',
+            'cat <<"SCRIPT" script.sh && bash -e script.sh\n'
```

Comment:
Bug: `cat: script.sh: No such file or directory` => `cat` tries to concatenate both the here-document and `script.sh`. I don't believe this is the right approach, for now.

Reply:
This should be `cat <<"SCRIPT > script.sh && bash -e script.sh`